### PR TITLE
chore: add service ulimit docs

### PR DESCRIPTION
**Pull request by meinenec**
adds docs for https://github.com/go-vela/types/pull/190

### CODE REVIEW

**Review by jbrockopp — DISMISSED** [review]
LGTM

File: content/reference/yaml/services.md [hunk 1]
```diff
@@ -22,11 +22,12 @@ services:
 | Tag           | Required | Type            | Description                                                     |
 |---------------|----------|-----------------|-----------------------------------------------------------------|
 | `name`        | Y        | string          | Unique identifier for the container in the pipeline             |
-| `image`       | Y        | []string        | Docker image used to create an ephemeral container                 |
+| `image`       | Y        | []string        | Docker image used to create an ephemeral container              |
 | `pull`        | N        | string          | Declaration to configure if and when the Docker image is pulled |
 | `environment` | N        | map || []string | Variables to inject into the container environment              |
 | `entrypoint`  | N        | []string        | Commands to execute inside the container                        |
 | `ports`       | N        | string          | List of ports to map for the container in the pipeline          |
+| `ulimits`     | N        | string          | Set the user limits for the container.                          |
```
Comment (wass3r) [inline content/reference/yaml/services.md]: i think this is `[]string` or `[]Ulimit` (though not sure how to depict "Ulimit" here), right?

ref: https://github.com/go-vela/types/blob/v0.8.2/yaml/ulimit.go#L51

Comment (wass3r) [reply content/reference/yaml/services.md]: probably throw something similar to what we put for `environment` in there.. `[]map || []string`?

**Review by wass3r — CHANGES_REQUESTED** [review]
see comment

File: content/reference/yaml/services.md (hunk 1 above)
Comment (meinenec) [reply content/reference/yaml/services.md]: ahh I think you're right, I'll update that and make a separate PR for steps (where I copied that from)
https://github.com/go-vela/docs/blob/master/content/reference/yaml/steps.md#tags

Looks like there's a couple fields that can be fixed in steps.

Comment (meinenec) [reply content/reference/yaml/services.md]: I thought about this a little and don't think `[]map` is valid, but `[]string` and `[]Ulimit` are.
Looking at how we document the [steps array for stages](https://go-vela.github.io/docs/reference/yaml/stages/#tags), it looks like we just use `[]string` so I'll just document it like that for now. Happy to be wrong and change it if you'd like 😄

**Review by wass3r — APPROVED** [review]

**Review by jbrockopp — APPROVED** [review]
LGTM

**Pull request merged**